ontology/yaml/resources/HVAC/entity_types/AHU.yaml

```diff
@@ -5317,3 +5317,19 @@ AHU_BSPC_DX4SC_ECON_EFSS_EFVSC_SFSS_SFVSC_SSPC_RHHC_NONCANONICAL_1:
   - heating_water_valve_percentage_command
   - min_zone_air_relative_humidity_sensor
   - min_zone_air_relative_humidity_setpoint
+
+AHU_SFSS_SFVSC_EFSS_CHWSC_RTM_FDPM2X_SSPC_OAFC_HTSC:
```

Comment:
Expect some sort of dehum control on this unit - need to determine exact sequence of ops for that

Reply:
Dehumidification uses zone air dewpoint compared to dewpoint setpoint.  If the zone is higher, chilled water valve is opened to 100% and the electric heating coil is modulated to reheat the air

Reply:
How can a multi-zone AHU control to a single zone sensor? Is it actually the return air dewpoint sensor?